include/zephyr/pm/state.h

```diff
@@ -209,6 +219,7 @@ struct pm_state_info {
 		.substate_id = DT_PROP_OR(node_id, substate_id, 0),	       \
 		.min_residency_us = DT_PROP_OR(node_id, min_residency_us, 0),  \
 		.exit_latency_us = DT_PROP_OR(node_id, exit_latency_us, 0),    \
+		.pm_device_disabled = DT_PROP_OR(node_id, zephyr_pm_device_disabled, true),    \
```

Comment:
the _OR will set `pm_device_disabled` to `true` when `zephyr,pm-device-disabled` is not set.
Since `zephyr,pm-device-disabled` is a boolean, you can use:
```suggestion
		.pm_device_disabled = DT_PROP(node_id, zephyr_pm_device_disabled),    \
```
which will set `pm_device_disabled` to true when `zephyr,pm-device-disabled` is present and vice versa.